diff -ur a1/conf/acng.conf.in b1/conf/acng.conf.in
--- a1/conf/acng.conf.in	2021-03-08 23:27:10.000000000 +0200
+++ b1/conf/acng.conf.in	2021-05-18 23:05:45.190944432 +0300
@@ -391,7 +391,7 @@
 # Default: guessed on startup by looking for explicit mention of apt-cacher-ng
 # in /etc/hosts.allow or /etc/hosts.deny files.
 #
-# UseWrap: 0
+UseWrap: 0
 
 # If many machines from the same local network attempt to update index files
 # (apt-get update) at nearly the same time, the known state of these index file
